### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by MEGA-Dawg68** [SethBurkart123/EvenBetterSEQTA#17] [Ideas]
I am going to say, what you did with the images is awesome! I had an idea with the themes, could BetterSEQTA+ change the colours/gradients to match the image? Also with the assesments and courses, it is see through and it is hard to see with the images. 
Over all, it is looking amazing

**Comment by SethBurkart123** [comment]
That would be interesting... I'd have to look at how we might be able to do that.
